src/main/java/org/broadinstitute/dsm/jobs/PubSubLookUpJob.java

```diff
@@ -0,0 +1,92 @@
+package org.broadinstitute.dsm.jobs;
+
+import com.google.cloud.pubsub.v1.AckReplyConsumer;
+import com.google.cloud.pubsub.v1.MessageReceiver;
+import com.google.cloud.pubsub.v1.Subscriber;
+import com.google.gson.Gson;
+import com.google.pubsub.v1.ProjectSubscriptionName;
+import com.google.pubsub.v1.PubsubMessage;
+import org.apache.commons.lang3.StringUtils;
+import org.broadinstitute.ddp.db.SimpleResult;
+import org.broadinstitute.dsm.model.birch.TestBostonResult;
+import org.quartz.Job;
+import org.quartz.JobExecutionContext;
+import org.quartz.JobExecutionException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+import static org.broadinstitute.ddp.db.TransactionWrapper.inTransaction;
+
+public class PubSubLookUpJob implements Job {
+    private static final Logger logger = LoggerFactory.getLogger(PubSubLookUpJob.class);
+
+    @Override
+    public void execute(JobExecutionContext context) throws JobExecutionException {
+        String projectId = "hl7-dev-274911";
+        String subscriptionId = "pegah-results-sub";
+
+        try{
+            ProjectSubscriptionName subscriptionName =
+                    ProjectSubscriptionName.of(projectId, subscriptionId);
+
+            // Instantiate an asynchronous message receiver.
+            MessageReceiver receiver =
+                    (PubsubMessage message, AckReplyConsumer consumer) -> {
+                        // Handle incoming message, then ack the received message.
+                        System.out.println("Id: " + message.getMessageId());
+                        System.out.println("Data: " + message.getData().toStringUtf8());
+                        writeResultsIntoDB(message);
+                        consumer.ack();
+                    };
+
```

Comment:
How often does this run, what's the cron schedule right now? Do we want to setup a long-running listener rather than spinning up/down a subscriber through a job?

Reply:
Good question, I run it every 30 minutes now, but we need to figure it out based on how "live" they want the results. In the current configuration,  does this mean we have to set the messages on the pubsub's topic to also have a 30 minute ack deadline time?

Reply:
I'd prefer  a long-term listener instead of startting/stopping subscribers inside  of scheduled  jobs.  It's more consistent with  our  other pubsub  methods.

We  can flow control this  with a single thread as well.